modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryRelatedObjectsResourceTest.java

```diff
@@ -815,7 +1045,10 @@ private int _invokeGetHttpCode(String location) throws Exception {
 		return response.getResponseCode();
 	}
 
-	private UserAccount _randomUserAccount() throws Exception {
+	private UserAccount _randomUserAccount(
+			String userExternalReferenceCode, String userEmailAdress)
+		throws Exception {
```

Comment:
🤔 🤔 🤔 🤔 
If you have to provide some values it is not random anymore, right? 😅 
AFAIK, a pattern that we use in other APIs is use the `randomEntity` method and after the creation setting the values that we want to have. For example:
```
UserAccount userAccount = new _randomUserAccount();
userAccount.setEmailAddress(yourEmailAddress)
userAccount.setExternalReferenceCode(yourExternalReferenceCode)
...
```

Reply:
Make sense @LuismiBarcos!!